logstash-core/spec/logstash/plugin_spec.rb

```diff
@@ -7,6 +7,16 @@
 require "logstash/filters/base"
 require "support/shared_contexts"
 
+class CustomFilterDeprecable < LogStash::Filters::Base
+  config_name "simple_plugin"
+    config :host, :validate => :string
+
+    def register
+      puts "Register invoked"
+      @deprecation_logger.deprecated "Deprecated feature {}", "teleportation"
```

Comment:
I suggest we generally use parens around arguments when calling a method. 
```
@deprecation_logger.deprecated("Deprecated feature {}", "teleportation")
```

Reply:
styleguide agrees :) https://github.com/rubocop-hq/ruby-style-guide#method-invocation-parens